apps/api_accounts/lib/api_accounts/changeset.ex

```diff
@@ -288,6 +288,26 @@ defmodule ApiAccounts.Changeset do
     end
   end
 
+  @doc """
+  Ensure email address is downcase and trimmed.
+
+  ## Examples
+
+      format_email(changeset, :email)
+
+  """
+  @spec format_email(t, atom) :: t
+  def format_email(%Changeset{} = changeset, field) do
```

Comment:
question: the previous version only formatted the e-mail address if the changeset was otherwise valid. Do we want to keep that behavior?

Reply:
> question: the previous version only formatted the e-mail address if the changeset was otherwise valid. Do we want to keep that behavior?

This is intentionally changed behavior. The email validator library rejects the input that we used to accept and clean up (`Nstein@domain.com ` => `nstein@domain.com`). This function now serves as a pass that takes place prior to validation to allow for those previously-acceptable inputs. 

We could also just not allow these inputs by removing the format email step entirely.